libcontainer/rootfs_linux.go

```diff
@@ -180,18 +180,22 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 			if err != nil {
 				return err
 			}
+			relDir, err := filepath.Rel(mm.Root, dir)
```

Comment:
If I am reading the code correctly, this doesn't have any effect unless we are planning to set Root to something else.